modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/tails.dm

```diff
@@ -4,69 +4,68 @@
 	organ_type = /obj/item/organ/external/tail
 	icon = 'modular_skyrat/master_files/icons/mob/sprite_accessory/tails.dmi'
 	special_render_case = TRUE
-	special_icon_case = TRUE
 	special_colorize = TRUE
 	relevent_layers = list(BODY_BEHIND_LAYER, BODY_FRONT_LAYER)
 	genetic = TRUE
-	/// A generalisation of the tail-type, e.g. lizard or feline, for MODsuit or other sprites
-	var/general_type
 	/// Can we use this tail for the fluffy tail turf emote?
 	var/fluffy = FALSE
 
-/datum/sprite_accessory/tails/get_special_render_state(mob/living/carbon/human/wearer)
-	// MODsuit tail spriting
-	if(general_type && wearer.wear_suit && istype(wearer.wear_suit, /obj/item/clothing/suit/mod))
-		if(wearer.back && istype(wearer.back, /obj/item/mod/control)) // If this fails, honestly, something's really wrong, but just to be safe...
-			var/obj/item/mod/control/modsuit_control = wearer.back
-			var/datum/mod_theme/mod_theme = modsuit_control.theme
-			if(mod_theme.modsuit_tail_colors)
-				return "[general_type]_modsuit"
+/datum/sprite_accessory/tails/is_hidden(mob/living/carbon/human/wearer, obj/item/bodypart/HD)
+	var/list/used_in_turf = list("tail")
```

Comment:
should this be a var on the obj itself?

Reply:
because i'm also using this for `organ/external/tail/can_draw_on_bodypart`, no. however the PR which its related to (turf emotes) is a spaghetti mess and should be refactored one day.